src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/service/CertifiedCopyItemService.java

```diff
@@ -62,10 +68,30 @@ public CertifiedCopyItem createCertifiedCopyItem(final CertifiedCopyItem certifi
         } else {
             certifiedCopyItem.getData().setPostalDelivery(Boolean.FALSE);
         }
+        certifiedCopyItem.getData().setKind("item#certified-copy");
```

Comment:
item#certified-copy is already defined as a constant on line 27, couldn't we use this instead of using a string literal here?

Reply:
Thanks. That one got away. Fixed now.